package.json

```diff
@@ -42,5 +42,8 @@
     "*.js": "eslint",
     "*.cjs": "eslint",
     "*.css": "stylelint"
+  },
+  "dependencies": {
+    "@adobe/rum-distiller": "^1.4.0"
```

Comment:
I don't think we need to list that here.

Reply:
I added it because otherwise the linting in the "Run tests and linting / build (pull_request)" failed (complains about not being able to find the dependency). Now that you point it out, it should be a dev dependency for sure but otherwise, do you have a better suggestion to make the linting work?

Reply:
Then it's ok.

Reply:
I moved it into the devDependencies at least